easybuild/easyconfigs/r/ROOT/ROOT-6.10.08-foss-2017b-Python-2.7.14.eb

```diff
@@ -38,6 +38,11 @@ dependencies = [
 configopts = '-Dxrootd=OFF -Dmysql=OFF -Dkrb5=OFF -Dodbc=OFF '
 configopts += '-Doracle=OFF -Dpgsql=OFF -Dqt=OFF '
 configopts += '-Dunuran=ON -Dtable=ON -Dexplicitlink=ON -Dminuit2=ON '
-configopts += '-Droofit=ON'
+configopts += '-Droofit=ON '
+
+# Standard ROOT build doesn't account for the fact that CFITSIO could be linked against cURL.
+# Official bug: https://sft.its.cern.ch/jira/browse/ROOT-9755
+# If CFITSIO is linked against cURL, this results in undefined reference errors during builds unless setting:
+configopts += '-DCMAKE_CXX_FLAGS="-Wl,--as-needed -lcurl" '
```

Comment:
@casparvl Please change this to the following, to avoid that `$CXXFLAGS` gets ignored completely...

```python
configopts += '-DCMAKE_CXX_FLAGS="$CXXFLAGS -Wl,--as-needed -lcurl" '
```

Reply:
Finally done... =)